homeassistant/components/deconz/device_trigger.py

```diff
@@ -417,7 +417,15 @@ async def async_validate_trigger_config(hass, config):
         or device.model not in REMOTES
         or trigger not in REMOTES[device.model]
     ):
-        raise InvalidDeviceAutomationConfig
+        if not device:
```

Comment:
Couldn't the main if be split to not need to put multiple layers of if statements?

Reply:
It could, sure. Matter of taste which way one wants to see it -- keeping the main `if` separate makes it stand out better from the rest of the code, which is why I did it this way.

Reply:
Allright. Let's keep it.